CRM/Report/Form.php

```diff
@@ -596,20 +595,8 @@ public function preProcessCommon() {
     if ($this->_id) {
       $this->assign('instanceId', $this->_id);
       $params = ['id' => $this->_id];
-      $this->_instanceValues = [];
-      CRM_Core_DAO::commonRetrieve('CRM_Report_DAO_ReportInstance',
```

Comment:
This line populates $this->_instanceValues - which is actually used in several other places so without this you don't load the formValues a few lines later.

- I don't know if this review will put it in the meh-scope-blowout category but  I would be included to have a function

loadInstanceValues()
- which calls apiv4 to load the values - or just does the above and

getInstanceValue($value);

which retrieves a value - calling ^^ if need be.